monitoring/thing-monitor/package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "thing-monitor",
+    "version": "1.0.0",
+    "description": "Monitoring service for Things with notification support",
+    "main": "dist/main.js",
+    "scripts": {
+      "build": "tsc",
+      "start": "node dist/main.js",
+      "dev": "ts-node src/main.ts",
+      "test": "jest",
+      "test:monitor": "ts-node src/test.ts"
+    },
+    "dependencies": {
+      "@node-wot/binding-coap": "^0.8.0",
+      "@node-wot/binding-http": "^0.8.0",
+      "@node-wot/binding-modbus": "^0.8.0",
+      "@node-wot/binding-mqtt": "^0.8.0",
+      "@node-wot/core": "^0.8.0",
```

Comment:
Maybe we can upgrade to ^0.9.0 ?

We should do the same in other places as well...
see https://github.com/eclipse-thingweb/test-things/issues/66